src/System.Private.ServiceModel/tests/Scenarios/Contract/Service/ServiceContractTests.cs

```diff
@@ -233,6 +234,191 @@ public static void ServiceContract_Call_Operation_With_MessageParameterAttribute
         Assert.True(errorBuilder.Length == 0, string.Format("Test Scenario: ServiceContract_Call_Operation_With_MessageParameterAttribute FAILED with the following errors: {0}", errorBuilder));
     }
 
+    // End operation includes keyword "out" on an Int as an arg.
+    [Fact]
+    [OuterLoop]
+    public static void ServiceContract_TypedProxy_AsyncEndOperation_IntOutArg()
+    {
```

Comment:
It would be a good idea to have synchronous versions of these.  It will exercise a different code path in packing and unpacking the out/ref parameters in WCF client code.